### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by abdellani**

### CODE REVIEW

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  display: flex;
+  margin-bottom:0px;
+}
+
+.navbar > li{
+  margin-bottom:0px;
+  text-transform: uppercase;
+  border: 1px solid black;
+  border-bottom: none;
+  padding: 15px;
```
Comment (JamesOkunlade) [inline dist/style.css]: consider adding `cursor: pointer;` to this selector.

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      'Dish name 4': 10,
+      'Dish name 5': 10,
+      'Dish name 6': 10,
+      'Dish name 7': 10,
+      'Dish name 8': 10,
+    },
+  },
+};
+
+function menu() {
```
Comment (JamesOkunlade) [inline src/menu.js]: This project requires you to use ES6 features. A `module` should be used here.

File: src/initial-page-load.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import createElement from './helpers';
+
+function initlizePage() {
```
Comment (JamesOkunlade) [inline src/initial-page-load.js]: An arrow function should be used here instead.

File: src/helpers.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function createElement(type, value = null) {
```
Comment (JamesOkunlade) [inline src/helpers.js]: An arrow function should be used here instead.

File: src/contact.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import createElement from './helpers';
+
+function contact() {
```
Comment (JamesOkunlade) [inline src/contact.js]: This contact should be a `module`

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hi @abdellani This project works perfectly :100:  I would have given it a pass if ES6 features were not important to this project. :disappointed_relieved:

This project comes after the introduction of the ES6 `class` keyword, ES6 `modules`, and `webpack`. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using `webpack`. Please refer to the readings preceding this project if you need to and try to implement different sections with the taught principle. 

You will need to use `arrow functions` rather than regular functions, use JavaScript `module` where you have to. I made some comments in the code which should help you. :sunglasses:

You should definitely get a pass after implementing these changes.:v:

**Status: `changes required`**

Happy coding!

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Oops!:grimacing: Something happened to the linter check.:weary: No check is being performed. Please the configuration again and request for one last re-review. :ok_hand:

Nice job implementing the changes I requested earlier though. Really nice 😎 

**Status: `changes required`**

**Comment by abdellani** [comment]
@JamesOkunlade 
I didn't touch the linter configuration files, you can check that on the new commits.

**Review by JamesOkunlade — APPROVED** [review]
### Haha 😁 It's a well deserved 👍  

Please go ahead and merge.

**Status: `Approved`**

**Good luck with your future projects 😎**

**Pull request merged**